src/utils/helpers.test.js

```diff
@@ -0,0 +1,16 @@
+import { getMovieYear, shuffle } from './helpers';
+
+describe('getMovieYear', () => {
+  it('should properly extract the year from 2018-03-30', () => {
+    expect(getMovieYear('2018-03-30')).toMatchSnapshot();
```

Comment:
As for me, snapshot here is redundant. Test is not only for testing. It is documentation also. Much easier to read test without snapshot. So prefer to use it only when you need to test equality with a big object/array or for rendered components
`expect(getMovieYear('2018-03-30')).toBe('2018');`

Reply:
fixed, thank you